src/api/teams.spec.ts

```diff
@@ -0,0 +1,44 @@
+import request from 'supertest';
+import app from 'app';
+
+describe("teams", () => {
+    it('sanity test', () => {
+        expect(true).toBe(true);
+    })
+
+    it("should respond with a 200 status code", async () => {
+        const response = await request(app).get("/teams");
+        expect(response.statusCode).toBe(200);
+    })
+
+    it("should get a teams by its email and name", async () => {
+        const response = await request(app).get("/teams/1");
+        expect(response.body).toStrictEqual({
+            email: "Amos38@gmail.com",
+            name: "Sharon Mills"
+        });
+    })
+
+    it("should post create a teams by its name, email and pass", async () => {
```

Comment:
Replace `should post create` with `should create`.
Replace `pass` with `password`.

Reply:
done.

Reply:
Replace should post create with should create.